Files/UserControls/NavigationToolbar.xaml.cs

```diff
@@ -76,6 +76,68 @@ public sealed partial class NavigationToolbar : UserControl, INavigationToolbar,
 
         public event EventHandler RefreshRequested;
 
+        public event EventHandler RefreshWidgetsRequested;
+
+        #region YourHome Widgets
```

Comment:
Can you remove the region?

Reply:
I'd insist on not removing region there since other properties (for buttons) also have their #region